blocks/v2-custom-form/v2-custom-form.js

```diff
@@ -81,6 +82,7 @@ async function submissionFailure() {
 
 // callback
 window.showResult = function showResult(json) {
+  console.log(json);
```

Comment:
To improve the use of these console logs, you can use a better way to show info, styling the logs:
`console.log('%cTriggering feedback response:', 'color:cyan', { json });`
OR
Remove the console logs